runtime/onert/core/src/util/shapeinf/Tile.cc

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+ir::Shape inferTileShape(const ir::Shape &in_shape, const int32_t *multiplier)
+{
+  // assert(in_shape.rank() == multiplier.rank());
+  ir::Shape new_Shape(in_shape.rank());
+
+  for (int i = 0; i < in_shape.rank(); ++i)
+  {
+    assert(multiplier[i]); // multiplier[i] shuld not be 0.
+    new_Shape.dim(i) = in_shape.dim(i) * multiplier[i];
+  }
+  return new_Shape;
+}
+
+void StaticInferer::visit(const ir::operation::Tile &op)
+{
+  const auto input_idx{op.getInputs().at(ir::operation::Tile::Input::INPUT)};
+  const auto &input = _operands.at(input_idx);
+
+  const auto multiplier_idx{op.getInputs().at(ir::operation::Tile::Input::MULTIPLES)};
+  const auto &multiplier = _operands.at(multiplier_idx);
+
+  const auto output_idx = op.getOutputs().at(0);
+  ir::Operand &output = _operands.at(output_idx);
+
+  if (input.info().isDynamic())
```

Comment:
please refer to https://github.com/Samsung/ONE/pull/1838#discussion_r434954225. I think we should check `multiplier`. `multiplier` can be the output of another op.